test/actors.test.js

```diff
@@ -208,6 +220,7 @@ describe('Actor methods', () => {
             const data = { id: runId, actId: actorId, status: 'SUCCEEDED' };
             const body = { data };
             const waitSecs = 1;
+            const maxItems = 100;
```

Comment:
is there a unit test that also works without `maxItems`? To verify that this param is really optional.